packages/pyright-internal/src/tests/fourslash/completions.params.fourslash.ts

```diff
@@ -1,10 +1,26 @@
 /// <reference path="fourslash.ts" />
 
 // @filename: test.py
+//// from typing import TypedDict, Unpack
+////
+//// class Movie(TypedDict):
+////     key1: str
+////     key2: int
+////
 //// def method(param1=None, param2='active', param3=None):
 ////     pass
 ////
 //// met/*marker1*/hod   /*marker2*/ (    /*marker3*/      param2 = 'test')
+////
+//// def method2(param1: int, **kwargs: Unpack[Movie]):
+////     pass
+////
+//// method2(p/*marker4*/, k/*marker5*/)
+////
+//// def method3(param1: int, **kwargs: Any):
+////     pass
+////
+//// method3(p/*marker6*/, k/*marker7*/)
```

Comment:
Looks like `marker6` isn't verified anywhere. Is that intentional?

Reply:
I should add a verification for this one too.